.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright 2023 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: cut-release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+permissions: read-all
```

Comment:
```suggestion
permissions: {}
```
I think we can just clear these as you are already setting required permissions in each job

Reply:
you dont need to set this in the level, you we setting for each job below, we can remove this

Reply:
setting the empty dictionary drops all permissions, otherwise you get the organisation/project default permissions; either `contents: read` or `permissions: write-all` (all scopes), depending on organisation and project configuration.

Reply:
we have set the read permission globally

Reply:
Added a comment about global read permissions. Didn't know that! Thank you